pivy-importer/src/main/groovy/com/linkedin/python/importer/ImporterCLI.java

```diff
@@ -94,12 +94,17 @@ private static void importPackages(CommandLine line, File repoPath) {
                 throw new IllegalArgumentException(errMsg);
             }
 
-            artifactDownloader.getProcessedDependencies().addAll(processedDependencies);
-            artifactDownloader.download();
-            processedDependencies.addAll(artifactDownloader.getProcessedDependencies());
+            pullDownPackageAndDependencies(artifactDownloader);
         }
     }
 
+    public static void pullDownPackageAndDependencies(DependencyDownloader artifactDownloader) {
+        Set<String> processedDependencies = new HashSet<>();
```

Comment:
Should you be recreating this set every time you call this function? Seems like maybe it should live in `importPackages()`.

Reply:
This is a good point! fixed.